docs/solidity-by-example.rst

```diff
@@ -454,13 +446,11 @@ high or low invalid bids.
             onlyBefore(revealEnd)
         {
             uint length = bids[msg.sender].length;
-            if (
-                _values.length != length ||
-                _fake.length != length ||
-                _secret.length != length
-            ) {
-                throw;
-            }
+            require(
+                _values.length == length &&
```

Comment:
I don't think we should do that...
I will change it into three separate requires for readability.